editor/find_in_files.cpp

```diff
@@ -221,16 +221,16 @@ float FindInFiles::get_progress() const {
 
 void FindInFiles::_scan_dir(String path, PoolStringArray &out_folders) {
 
-	DirAccessRef *dir = DirAccessRef::open(path);
-	if (dir == NULL) {
+	DirAccessRef dir = DirAccess::open(path);
+	if (dir) {
 		print_verbose("Cannot open directory! " + path);
 		return;
 	}
 
-	dir->list_dir_begin();
+	dir.list_dir_begin();
```

Comment:
This won't compile either. It has to remain `->`. DirAccessRef is a wrapper to pointer, you don't need to change this. Please try to compile Godot to be sure your change works.